components/brave_news/browser/resources/internals.tsx

```diff
@@ -29,13 +28,15 @@ const Grid = styled(Variables)`
 `
 
 function App() {
+  const { result: exportData } = usePromise(getExportData, [])
   return <Grid data-theme="dark">
```

Comment:
Just curious where this `data-theme` is handled? wanted to learn :)

Reply:
Found it! - https://github.com/brave/leo?tab=readme-ov-file#colors

Reply:
Yeah, basically it sets a Nala variables to their dark mode version under this node :smile: There's an equivalent `data-theme=light`